sdk/src/test/java/com/hedera/hashgraph/sdk/HbarTest.java

```diff
@@ -83,7 +83,9 @@ void constructorWorks() {
 
     @Test
     void fromString() {
-        assertEquals(Hbar.fromString("1").toTinybars(), 100000000);
+        assertEquals(Hbar.fromString("1").toTinybars(), 100_000_000);
+        assertEquals(Hbar.fromString("ℏ 1").toTinybars(), 100_000_000);
+        assertEquals(Hbar.fromString("mℏ1").toTinybars(), 100_000);
```

Comment:
Lets require a space between the symbol and number as well. So `1ℏ` would not work, while `1 ℏ` will. A strict format is better than a lose one.